app/components/Project/ProjectForm.tsx

```diff
@@ -159,31 +137,13 @@ const ProjectForm: React.FC<Props> = (props) => {
           title: "Trade Name",
         },
       },
-      fundingStreamId: {
-        "ui:placeholder": "Select a Funding Stream",
-        "ui:widget": "select",
-        "ui:col-md": 12,
-        "bcgov:size": "small",
-      },
-      fundingStreamDescription: {
-        "ui:col-md": 12,
-        "ui:widget": "DisplayOnly",
-        "bcgov:size": "large",
-        "ui:options": {
-          text: `${
-            selectedFundingStream ? selectedFundingStream.node.description : ""
-          }`,
-          title: "Description",
-        },
-      },
       fundingStreamRfpId: {
-        "ui:placeholder": "Select a Funding Stream RFP Year",
-        "ui:widget": "select",
+        "ui:widget": "SelectParentWidget",
```

Comment:
Instead of having a completely generic widget here and passing its options via the `schema`, id rather have a `SelectFundingStreamRfpWidget`.

The `SelectFundingStreamRfpWidget` should use `useFragment`, so that the relay fragment it uses are declared in the same file where they are used. Right now `eslint` should show you warnings about some parts of the relay query not being used in this file.

`SelectFundingStreamRfpWidget` can definitely use a generic `SelectParentWidget` component, and do things such as parsing the list of funding streams and RFPs, making a valid schema to use by `SelectParentWidget`.

The `query` that the fragment will need can be passed via the `formContext`.